core/tabs/security/tab_data.toml

```
@@ -5,3 +5,8 @@ name = "Firewall Baselines (CTT)"
 description = "Developed to ease iptables firewall configuration, UFW provides a user friendly way to create an IPv4 or IPv6 host-based firewall.\nThis command installs UFW and configures UFW based on CTT's recommended rules.\nFor more information visit: https://christitus.com/linux-security-mistakes"
 script = "firewall-baselines.sh"
 task_list = "I SS"
+
+[[data.preconditions]]
+matches = false
+data = "command_exists"
+values = [ "dnf" ]
```

```suggestion
values = [ "firewalld" ]
```

Why check the distro, rather than for the conflicting package itself?